doc/src/manual/faq.md

```diff
@@ -732,6 +732,27 @@ julia> @sync for i in 1:3
 1 Foo  Bar 2 Foo  Bar 3 Foo  Bar
 ```
 
+## Arrays
+
+### Zero-dimensional arrays
+
+Zero-dimensional arrays are arrays of the form `Array{T,0}`. They behave similar to scalars, but there are important differences. They deserve a special mention because they are a special case which makes logical sense given the generic definition of arrays, but might be a bit unintuitive at first. The following line defines a zero-dimensional array:
```

Comment:
Break lines at 92 chars.

Reply:
I don't understand. Do I need to do something?

Reply:
Done